DigitalLearningSolutions.Web/ViewModels/LearningMenu/PostLearningContentViewModel.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.LearningMenu
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Data.Models.PostLearningAssessment;
+    using DigitalLearningSolutions.Web.Helpers;
+    using Microsoft.Extensions.Configuration;
+
+    public class PostLearningContentViewModel
+    {
+        public int CustomisationId { get; }
+        public int SectionId { get; }
+        public string CourseTitle { get; }
+        public string SectionName { get; }
+        public string ContentSource { get; }
+        public List<int> Tutorials { get; }
```

Comment:
Does this need to be public?

Reply:
Looks like this was used in a test. Since we're indirectly testing the tutorials when we test HTML assessments, I could remove the specific tutorial list test?

Reply:
Yes please. Tests should only be testing the public part of a class 🙂